lib/emerald/nodes/Attributes.rb

```diff
@@ -6,10 +6,11 @@
 
 # The attributes hash for an element
 class Attributes < Node
-  def to_html(_)
-    # TODO: make this work
-    elements
-      .map(&:text_value)
-      .join(' ')
+  def to_html(_context)
+    output = ''
+    elements.each do |e|
+      output += " #{e.elements[0].text_value}=#{e.elements[2].text_value}"
```

Comment:
Not sure if the `label:rule` syntax works in nested parts of a grammar too. If it doesn't dw about it